### PR TITLE
chore: update project

### PR DESCRIPTION
**Pull request by morteza-jamali**
Code structure improved and **.npmignore** file added

### CODE REVIEW

**Comment by unional** [comment]
Hi, thanks for the PR.

What do you want to do here? It's a TypeScript project so most of your changes doesn't look right.

**Comment by morteza-jamali** [comment]
I want to minimize the size of project . when i installed path-equal package , i saw unnecessary files .

**Comment by unional** [comment]
I see. In that case, can you use the `files` property instead?
https://medium.com/@jdxcode/for-the-love-of-god-dont-use-npmignore-f93c08909d8d

And should not change the code to use function expression and default export.
Also, changing the package name and version may not help either. :)

**Comment by morteza-jamali** [comment]
Sorry about version and name changing .I forgot not to change the name and version of this branch . thank u about your help .

File: src/pathEqual.ts [hunk 1]
```diff
@@ -1,17 +1,23 @@
-export function pathEqual(actual: string, expected: string) {
-  if (actual === expected) return true
```
Comment (unional) [inline src/pathEqual.ts:2]: I would keep this here because the normalization takes time. If we can return early without doing it, it is better.

Comment (morteza-jamali) [reply src/pathEqual.ts:2]: That's ok .

File: .github/workflows/nodejs.yml [hunk 2]
```diff
@@ -87,8 +87,6 @@ jobs:
         uses: codacy/codacy-coverage-reporter-action@master
         with:
           project-token: ${{ secrets.CODACY_PROJECT_TOKEN }}
-        env:
```
Comment (unional) [inline .github/workflows/nodejs.yml:88]: Don't worry about the codacy. I'll try to fix it later on.

thx

**Review by unional — APPROVED** [review]
Thanks for the PR

**Pull request merged**